packages/shared/src/components/modals/UpvotedPopupModal.tsx

```diff
@@ -54,7 +54,10 @@ export function UpvotedPopupModal({
         <h3 className="font-bold typo-title3">Upvoted by</h3>
         <ModalCloseButton onClick={onRequestClose} />
       </header>
-      <section className="w-full relative flex-shrink h-full max-h-full overflow-auto">
+      <section
+        className="w-full relative flex-shrink h-full max-h-full overflow-auto"
+        aria-label={`List of ${queryKey[0]} with ID ${queryKey[1]}`}
```

Comment:
I don't think you want to use `aria-label` here. If I get your point right you want it for testing.
Then you need:
https://testing-library.com/docs/queries/bytestid/

Reply:
Ah, yes. That is the more appropriate one. Thank you for the tip! 💯